TypeScript.py

```diff
@@ -455,21 +458,31 @@ def sendReplaceChangesForRegions(view, regions, insertString):
         endLocation = getLocationFromPosition(view, region.end())
         cli.service.change(view.file_name(), location, endLocation, insertString)
 
+def recvReloadAck(reloadResp):
```

Comment:
What does "Ack" mean? If you mean "async", "recv_reload_async" maybe better. For new functions / methods, we should try use [python convention](https://www.python.org/dev/peps/pep-0008/#naming-conventions) (lower_case_with_underscores.)